packages/asar.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,14 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'asar'
+$category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+$arguments = '--help'
+
+try {
+    # "--no-update-notifier" removes the npm funding notice that may cause an error
```

Comment:
@sara-rn could you please add this comment to [common](https://github.com/mandiant/VM-Packages/blob/d7e532fe2193e2d2a51f58bf5c6bf8eb6b12044a/packages/common.vm/tools/vm.common/vm.common.psm1#L508) too? :pray:  It is ok in this or a separate PR.